pylint_odoo/test_repo/broken_module/tests/test_model.py

```diff
@@ -0,0 +1,8 @@
+from odoo.tests.common import TransactionCase
+
+
+class BrokenModel(TransactionCase):
```

Comment:
This is a test file, so it should not have `_inherit` and it should be named as `TestModel`

Reply:
Done